website/routes.py

```diff
@@ -197,6 +198,13 @@ def make_url_map(app):
             OsfWebRenderer('public/pages/meeting_landing.mako'),
         ),
 
+        Rule(
+            '/api/v1/citation_styles/',
```

Comment:
The first `process_rules` call seems to deal with site-wide routes - things that don't deal with a specific object. I put it there because there is another `/api/v1` route above it for conferences.

There's no reason it shouldn't be moved - but I didn't really see a better place for it :).